git-merge-conflict.md

```diff
@@ -0,0 +1,52 @@
+# Merge Conflict Carnival
+## Overview
+This year the Ada staff has generously agreed to contribute their culinary skills to a coding bootcamp booth offering food & info pamphlets at the [Georgetown Carnival](http://www.georgetowncarnival.com/). It's the day before the festivities kick off and _someone's_ Cavalier King Charles Spaniel has done his best to shred all of their recipes! Oh no! Can you help Ada staff put everything back together and save Carnival?
+
+In this activity you'll be working with your fellow Adies to put together multiple scraps from a single recipe using Git. You'll accomplish this by successively merging in content from each individuals' copy of the recipes repo. Once all the of merges are finished (and any merge conflicts resolved) you'll have complete recipe document which can be submitted to the original repository with a pull request.
+
+## Instructions
+### Baseline Setup
+Before we can start putting the recipe together we need to make sure that everyone in your group is working from the same starting point and can effectively contribute to the recipe building effort.
+
+1. **Split into groups**: Each table is marked with a particular group number and everyone at that table will be part of the group. There should be six people in each group.
```

Comment:
We can easily switch the number of recipes, or number of scraps per group (I have a script that randomly divides a recipe file into N scraps).

However my thinking was that this would be a good activity for the students to do in the auditorium with mixed-class groups, if we can fit that into the schedule.